.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: Main
+
+on:
+  push:
+    branches: [main]
+  schedule:
+    - cron: "0 8,20 * * *"
+
+jobs:
+  unit-tests:
+    name: Unit tests
+    uses: apple/swift-nio/.github/workflows/unit_tests.yml@main
+    with:
+      linux_5_9_enabled: false
+      linux_5_10_enabled: false
+      linux_6_0_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
+      linux_nightly_6_0_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
+      linux_nightly_main_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
+
+  integration-tests:
+    name: Integration tests
+    uses: apple/swift-nio/.github/workflows/swift_matrix.yml@main
+    with:
+      name: "Integration Tests"
+      matrix_linux_5_9_enabled: false
+      matrix_linux_5_10_enabled: false
+      matrix_linux_command: "./dev/integration_tests.sh"
```

Comment:
This job is failing because this script doesn't exist 🙃

Reply:
That's true! As far as I can tell this repo doesn't have integration tests beyond benchmarks so I've deleted this block.